### ISSUE TITLE
Global Location for Documentation Files

### ISSUE THREAD

**Issue by charlesmjm** [sphinx-doc/sphinx#1743]
Hello,

I'm trying to figure out if something is possible with Sphinx. I have 3 separate documentations atm that have been created using Sphinx with rst files. These 3 documentations use many of the same rst files. Is there a way that I can put all rst files in one central location and then have each documentation grab the files they require. The purpose for this is that if I have to update an rst file, I wouldn't have to go through each of the 3 documentations and make the changes.

I am aware of "..include::" but i found that when i use this with files that require images that they don't carry over and i get build errors saying the images are missing. 

If someone can give me an answer as to how to I can make this possible, an alternative, or if I am wasting my time, that would be awesome.

Thanks,

Charles

**Comment by keimlink** [comment]
Can you please add a short example of files you want to include? Can you also explain the structure of your documentation briefly?

**Comment by charlesmjm** [comment]
I've added a screen shot of the sphinx make, which shows that everything is not being written. I've also attached a screen shot of the html showing the image is absent. My rst and image files are located one directory up because I am using them for 3 Documentations and I am grabbing them with .. include::

![sphinxfail](https://cloud.githubusercontent.com/assets/11197696/6396415/ea418a70-bda3-11e4-887b-27cde71e1fee.PNG)

![nopicture](https://cloud.githubusercontent.com/assets/11197696/6396421/f47e479e-bda3-11e4-8a43-706b94ea733f.PNG)

Currently I have my 3 Documentation sphinx directories along with directories that contain .rst and image files that i want all 3 Documentations to use. Within each documentations source it has directories that contain specific .rst files to that documentation and also .rst files with the ..include:: function using a relative path to obtain the shared files.

![directories](https://cloud.githubusercontent.com/assets/11197696/6396675/647efede-bda5-11e4-928c-419f29c83f58.PNG)

I was wondering if there's a way to avoid the include entirely tho. I would like to have a single source location to store all of my .rst and image files for all 3 of my documentations to pull their files from yet still have 3 separate index and builds files if possible.

Thanks,

Charles
